src/main/app/invite-applicant2/InviteApplicant2PostController.ts

```diff
@@ -0,0 +1,24 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { ApplicationType } from '../case/definition';
+import { AppRequest } from '../controller/AppRequest';
+import { AnyObject, PostController } from '../controller/PostController';
+import { Form } from '../form/Form';
+
+@autobind
+export class InviteApplicant2PostController extends PostController<AnyObject> {
+  constructor(protected readonly form: Form) {
+    super(form);
+  }
+
+  public async post(req: AppRequest<AnyObject>, res: Response): Promise<void> {
+    const { saveAndSignOut, saveBeforeSessionTimeout, _csrf, ...formData } = this.form.getParsedBody(req.body);
+
+    if (formData.applicationType === ApplicationType.JOINT_APPLICATION) {
```

Comment:
maybe we can have this in the actual post controller, does not seem like much is going on here so it can be another if like how we do save and signout or timeout?

Reply:
As I needed custom post behaviour, Seb said this was the way to do it

Reply:
I had previously had it the way you are describing and it wasn't working for me

Reply:
You could change the `save` method in the `PostController` to be `protected` and that way you could bring the new method from the `PostController` here.